atmo/atmo_test.go

```diff
@@ -0,0 +1,185 @@
+package atmo
+
+import (
+	"bytes"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/suborbital/atmo/atmo/options"
+	"github.com/suborbital/vektor/vtest"
+)
+
+//curl -d 'my friend' localhost:8080/hello
+func TestHelloEndpoint(t *testing.T) {
+	atmo := atmoForBundle("../example-project/runnables.wasm.zip")
+
+	server, err := atmo.testServer()
+	if err != nil {
+		t.Error(err)
+		return
```

Comment:
This can be simplified to:
```suggestion
		t.Fatal(err)
```
(suggestion applies to all of the other occurrences)